### ISSUE TITLE
Prefer-template multiline 

### ISSUE THREAD

**Issue by BridgeAR** [eslint/eslint#9983]
* **ESLint Version:** 4.17
* **Node Version:** 10-pre
* **npm Version:** 5.6.0

This is pretty much reopening #3507.

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

```js
prefer-template: error
```

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
const foo = 123;
const message = `${foo}   1,\n`.repeat(10) +
    '...\n';
```

**What did you expect to happen?**

No error should be reported in such a case. In general it is not preferable to have multi line template strings even though using template strings in general is good.

**What actually happened? Please include the actual, raw output from ESLint.**

```
error  Unexpected string concatenation  prefer-template
```

**Comment by BridgeAR** [comment]
@Cellule @mathieumg this might be interesting for you as well. The same applies to https://github.com/eslint/eslint/issues/9984.
